coordinator/src/main/scala/filodb.coordinator/queryplanner/CompositePlanner.scala

```diff
@@ -17,17 +17,19 @@ import filodb.query.exec._
 class CompositePlanner(dsRef: DatasetRef,
                        schemas: Schemas,
                        shardMapperFunc: => ShardMapper,
-                       //                  downsampleMapperFunc: => ShardMapper,
+                       downsampleMapperFunc: => ShardMapper,
                        failureProvider: FailureProvider,
+                       earliestRawTimestamp: => Long,
```

Comment:
How do we obtain this?

Reply:
Typically, `System.currentTimeMillis - rawDataRetentionMillis`